NodeChrome/generate_config

```diff
@@ -10,10 +10,11 @@ cat <<_EOF
       "browserName": "chrome",
       "maxInstances": $NODE_MAX_INSTANCES,
       "seleniumProtocol": "WebDriver",
+	  "httpProxy": "s1firewall:8080",
       "applicationName": "$NODE_APPLICATION_NAME"
     }
   ],
-  "proxy": "org.openqa.grid.selenium.proxy.DefaultRemoteProxy",
+  "host": $NODE_IP,
```

Comment:
$NODE_IP isn't added to NodeBase/Dockerfile therefore there is no way to access this variable.

Also why are you removing the default proxy